src/devices/ServoMotor/ServoMotor.cs

```diff
@@ -0,0 +1,133 @@
+        /// <param name="maximumPulseWidthMicroseconds">The maxnimum pulse width, in microseconds, that represent an angle for maximum angle.</param>
+        public ServoMotor(
+            PwmChannel pwmChannel,
+            int maximumAngle = 180,
+            int minimumPulseWidthMicroseconds = 1_000,
+            int maximumPulseWidthMicroseconds = 2_000)
+            : this(maximumAngle, minimumPulseWidthMicroseconds, maximumPulseWidthMicroseconds)
+        {
+            _pwmChannel = pwmChannel;
+        }
+
+        /// <summary>
+        ///  Initializes a new instance of the <see cref="ServoMotor"/> class.
+        /// </summary>
+        /// <param name="chip">The PWM chip number used to control the servo motor.</param>
+        /// <param name="channel">The PWM channel number used to control the servo motor.</param>
+        /// <param name="maximumAngle">The maximum angle the servo motor can move.</param>
+        /// <param name="minimumPulseWidthMicroseconds">The minimum pulse width, in microseconds, that represent an angle for 0 degrees.</param>
+        /// <param name="maximumPulseWidthMicroseconds">The maxnimum pulse width, in microseconds, that represent an angle for maximum angle.</param>
+        public ServoMotor(
+            int chip,
+            int channel,
+            int maximumAngle = 180,
+            int minimumPulseWidthMicroseconds = 1_000,
+            int maximumPulseWidthMicroseconds = 2_000)
+            : this(maximumAngle, minimumPulseWidthMicroseconds, maximumPulseWidthMicroseconds)
+        {
+            _pwmChannel = PwmChannel.Create(chip, channel, 50);  // Default 50Hz
+        }
+
+        private static double CalculateServoAngleSlope(int maximumAngle, int minimumPulseWidthMicroseconds, int maximumPulseWidthMicroseconds)
+        {
+            if (maximumAngle < 0 || maximumAngle > 360)
+            {
+                throw new ArgumentOutOfRangeException(nameof(maximumAngle), maximumAngle, "Value must be between 0 and 360.");
+            }
+
+            if (minimumPulseWidthMicroseconds < 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(minimumPulseWidthMicroseconds), minimumPulseWidthMicroseconds, "Value must not be negative.");
+            }
+
+            if (maximumPulseWidthMicroseconds < minimumPulseWidthMicroseconds)
+            {
+                throw new ArgumentOutOfRangeException(nameof(maximumPulseWidthMicroseconds), maximumPulseWidthMicroseconds, $"Value must be greater than or equal to {minimumPulseWidthMicroseconds}.");
+            }
+
+            // slope = (y2 - y1) / (x2 - x1).  x1 is 0 in this case, so it is not subtracted.
+            double servoAngleSlope = (double)(maximumPulseWidthMicroseconds - minimumPulseWidthMicroseconds) / maximumAngle;
+            return servoAngleSlope;
+        }
+
+        /// <summary>
+        /// Starts the servo motor.
+        /// </summary>
+        public void Start() => _pwmChannel.Start();
```

Comment:
I feel like this should be in the constructor. Same we don't have `Thermometer.Start` which constructs I2cDevice or some other micro optimizations with unclear usage

Reply:
I thought about that when we worked on the new `PwmChannel`, where the signal automatically starts when creating the object.  However, users will want to start/stop on demand without having to dispose and recreate the object.  The signals can still be adjusted while it is enabled/started, but there will be times they need to stop the signal while working on things, make other adjustments (not just for the PWM signal) and start signals back.

Users could damage hardware and what they are controlling if not careful.  For example, they could use the default PwmChannel and not specific the frequency (400Hz) and duty cycle (50%).  Let's say they had it connected to throttle on remote control lawn mower.  If they created servo motor and it automatically started... bad things could happen 😄

Reply:
@shaggygi as I previously wrote: for PWM I don't think Start/Stop are justified because 0% fill is equivalent to not starting. In here 0% fill I think it might cause servo not to move in which case Start/Stop are justified: Start causes servo to start moving into the set position and Stop stops any movement. 0% would mean: move to the left vs Stop means don't move at all in which case this sounds justify - I just need to hear the confirmation that is the case 😄

Reply:
@krwq  Here are some other library examples that offer similar for start/stop.

https://www.arduino.cc/en/Reference/ServoAttach
https://rpi.science.uoit.ca/lab/servo/#example-of-controlling-a-servo-from-the-raspberry-pi